start_training.py

```diff
@@ -125,12 +125,12 @@ def parse_args(**kwargs):
         'scalar': config.scalar,
         'discrete_values': config.discrete_values,
     }
-    if config.model is None:
-        assert config.depth is not None and config.width is not None
-        model = create_model(config.depth, config.width, **model_kwargs)
-    else:
-        model_creator = getattr(f3_models, config.model)
-        model = model_creator(**model_kwargs)
+    # if config.model is None:
+    assert config.depth is not None and config.width is not None
```

Comment:
Please remove this line and instead set the depth and width arguments as `required` (in lines 61 and 62, see line 80 for an example) as they are now always required.